### DISCUSSION TITLE
提议

### DISCUSSION THREAD

**Discussion by RainbowTaro2022** [Meloong-Git/PCL#1702] [讨论]
PCL2在加载时没有小游戏，是不是感觉有点无聊了呢？
我想在PCL里加入小游戏，这个提议可以吗？

**Comment by Chiloven945** [comment]
加载游戏时需要很多的内存进行载入，这时候引入小游戏会占用较多内存影响游戏运行效率，而且这个功能的受用面积不广所以说不太可能会同意……
当然如果你有任何想法都可以提出来或者交一个issue

**Comment by MoYuan-CN** [comment]
有新的建议很好，你可以到 [这里](https://github.com/Hex-Dragon/PCL2/issues/new/choose) 开启一个新的提案！
BTW 就如楼上所说，这个功能的受用面积不广，不太可能制作
